### PR TITLE
Fix vulkan app crash issue

### PR DESCRIPTION
**Pull request by renchenglei**

### CODE REVIEW

**Comment by renchenglei** [comment]
@feijiang1 & @shyjumon-n
Can you help take a review of this changes? Thanks a lot!

**Review by feijiang1 — APPROVED** [review]

File: aosp_diff/preliminary/frameworks/native/03_0001-Inline-GetPhysicalDeviceProperties2-into-QueryPresen.patch [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+diff --git a/vulkan/libvulkan/driver.cpp b/vulkan/libvulkan/driver.cpp
+index d7fdab558669..cf774fd9b82b 100644
+--- a/vulkan/libvulkan/driver.cpp
++++ b/vulkan/libvulkan/driver.cpp
+@@ -979,6 +979,8 @@ VkResult EnumerateInstanceExtensionProperties(
+ void QueryPresentationProperties(
+     VkPhysicalDevice physicalDevice,
+     VkPhysicalDevicePresentationPropertiesANDROID* presentation_properties) {
++    ATRACE_CALL();
```
Comment (shyjumon-n) [inline aosp_diff/preliminary/frameworks/native/03_0001-Inline-GetPhysicalDeviceProperties2-into-QueryPresen.patch:44]: Is it intentional? will it impact performance ?

**Review by shyjumon-n — COMMENTED** [review]
Could you please check my comment

File: aosp_diff/preliminary/frameworks/native/03_0001-Inline-GetPhysicalDeviceProperties2-into-QueryPresen.patch (hunk 1 above)
Comment (renchenglei) [reply aosp_diff/preliminary/frameworks/native/03_0001-Inline-GetPhysicalDeviceProperties2-into-QueryPresen.patch:44]: Thanks @shyjumon-n for the comments. I pulled this changes from google, and didn't make any changes, the origin patch can be found at:
https://android-review.googlesource.com/c/platform/frameworks/native/+/1908993

**Review by feijiang1 — APPROVED** [review]

**Review by shyjumon-n — APPROVED** [review]
Google patch port. LGTM.

**Pull request merged**